python/pyspark/pandas/tests/test_series.py

```diff
@@ -998,6 +999,10 @@ def test_sort_values(self):
         self.assert_eq(psser, pser.sort_values())
         self.assert_eq(psdf, pdf)
 
+        psser.sort_values(inplace=True, ascending=False, ignore_index=True)
+        self.assert_eq(psser, pser.sort_values(ascending=False, ignore_index=True))
+        self.assert_eq(psdf, pdf)
```

Comment:
```suggestion
        psser.sort_values(inplace=True, ascending=False, ignore_index=True)
        pser.sort_values(inplace=True, ascending=False, ignore_index=True)
        self.assert_eq(psser, pser)
        self.assert_eq(psdf, pdf)
```

Reply:
Shall we be consistent with https://github.com/apache/spark/blob/master/python/pyspark/pandas/tests/test_series.py#L1028-L1033 instead? Thanks!

Reply:
Shall we put the comment here as well, then?

Reply:
Sounds good!